scripts/kubestellar

```diff
@@ -302,6 +305,21 @@ if [ $subcommand == stop ]; then
    exit 0
 fi
 
+# Check if kcp is running
+if [ "$local_kcp" == true ] && [ $(process_running kcp) != "running" ]; then
+    echo "kcp is not running - please start it ...."
+    exit 1
+fi
+
+let tries=1
+while ! ( kubectl get workspaces && kubectl ws root:compute ); do
```

Comment:
Do we have any test to verify that kubectl ws plugin is in the path ?

Reply:
well, this is testing that.

Reply:
OK. I meant distinctly. Fine for now, for the future we may want separate tests for better root cause analysis